sparkle/src/components/Citation.tsx

```diff
@@ -52,9 +52,13 @@ export function Citation({
           </a>
         )}
       </div>
-      <div className="s-text-sm s-font-bold s-text-element-900">{title}</div>
+      <Tooltip label={title} position="above">
```

Comment:
What is the tooltip for given that it just echos the title?

Reply:
Title is limited to 1 line